test/integration/middleware/core/test/index.test.js

```diff
@@ -161,67 +163,34 @@ function urlTests(_log, locale = '') {
     expect($('.title').text()).toBe('URLs A')
   })
 
-  it('throws when using URL with a relative URL', async () => {
-    const res = await fetchViaHTTP(
-      context.appPort,
-      `${locale}/urls/relative-url`
-    )
-    const json = await res.json()
-    expect(json.error.message).toContain('Invalid URL')
```

Comment:
Am I correct this is the case that error happened on a middleware invocation? I wonder what happens now.

Reply:
_discussed online_

@nkzawa correctly spot some tests that I should have not removed. Restoring them now.